packages/default-theme/components/SwProductDetails.vue

```diff
@@ -3,6 +3,7 @@
     <div class="product-details__mobile-top">
       <SwProductHeading
         class="product-details__heading"
+        :shippingFree="shippingFree"
```

Comment:
this could be refactored to pass only `product` as property, and other props may be computed inside

Reply:
refactored to one props: poduct